Source/Examples/ExampleLibrary/Utilities/PlotModelUtilities.cs

```diff
@@ -10,16 +10,48 @@
 namespace ExampleLibrary.Utilities
 {
     using System;
+    using System.Collections.Generic;
     using System.Linq;
+    using System.Reflection;
+
     using OxyPlot;
     using OxyPlot.Axes;
     using OxyPlot.Series;
 
+
     public static class PlotModelUtilities
     {
         private const string XAXIS_KEY = "x";
         private const string YAXIS_KEY = "y";
 
+        /// <summary>
+        /// Lists all XYAxisSeries from the core library that are NOT transposable
+        /// </summary>
+        private static readonly HashSet<Type> NonTransposableSeriesTypes = new HashSet<Type>
+        {
+            typeof(CandleStickAndVolumeSeries),
+            typeof(OldCandleStickSeries),
+        };
+
+        /// <summary>
+        /// Returns a value indicating whether a plot model is transposable.
+        /// </summary>
+        /// <param name="model">The plot model.</param>
+        /// <returns>True if the plot model in transposable; false otherwise.</returns>
+        public static bool IsTransposable(this PlotModel model)
+        {                   
+            return (model.Axes.Count > 0 || model.Series.Count > 0)
+                && model.Annotations.Count == 0
```

Comment:
could annotations also be transposed?

Reply:
Interesting question! I will open a separate issue for it.